core/primitives/src/epoch_manager.rs

```diff
@@ -153,6 +157,22 @@ impl AllEpochConfig {
         &self.chain_id
     }
 
+    /// Configures mocknet-specific features only.
+    fn config_mocknet(config: &mut EpochConfig, chain_id: &str, protocol_version: ProtocolVersion) {
+        if chain_id == near_primitives_core::chains::MOCKNET {
+            if checked_feature!("stable", StatelessValidationV0, protocol_version) {
+                // In production (mainnet/testnet) and nightly environments this setting is guarded by
+                // ProtocolFeature::StatelessnetShuffleShardAssignmentsForChunkProducers.
+                // (see config_validator_selection function). For pre-release environment such as mocknet,
+                // we enable it by default with stateless validation to exercise the codepaths for state sync more often.
```

Comment:
I was thinking about enabling it with stateless validation only, not always (even in existing stateful validation). This is why I check the stateless validation feature explicitly. But we can also do that independently because without single-shard tracking (which comes with stateless validation) it should not have any effect. what do you think?